INSTALL.md

```diff
@@ -102,80 +93,100 @@ http://lcdproc.org/hardware.php3 for details. Several
 different types of displays are now supported.
 
 
--- BUILDING LCDPROC -------------------------------------------------
+# BUILDING LCDPROC
 
---- Preparing a Git distro ---
+## Preparing a Git distro
 If you retrieved these files from the Git, you will first need to run:
-    sh ./autogen.sh
+```
+sh ./autogen.sh
+```
 
---- Configuration ---
+## Configuration
 The simplest way of doing it is with:
-    ./configure
+```
+./configure
+```
 But it may not do what you want, so please take a few seconds to type:
-    ./configure --help
-And read the available options, especially --enable-drivers
+```
+./configure --help
+```
+And read the available options, especially ```--enable-drivers```
 
---- Compilation ---
+## Compilation
 Run make to build the server and all clients
-    make
-
+```
+make
+```
 If you only want to compile the clients, you can omit to compile the
 server:
-    make clients
+```
+make clients
+```
 
 Similarly, if you only want to compile the server, you can omit to
 compile the clients:
-    make server
+```
+make server
+```
 
 Depending on your system, LCDproc will build in a few seconds to a
 few minutes. It's not very big.
 
 If you want to, you can install it (if you're root) by typing:
-    make install
+```
+make install
+```
 
 This will install the binaries and the man pages in the directory you
 specified in configure.
-You may have to copy the configuration file (LCDd.conf) to /etc
-(or /usr/local/etc) manually.
+You may have to copy the configuration file (```LCDd.conf```) to ```/etc```
+(or ```/usr/local/etc```) manually.
 
 
--- CREATING DISTRIBUTION SPECIFIC PACKAGES --------------------------
+# CREATING DISTRIBUTION SPECIFIC PACKAGES
 
 Instead of manually configuring, compiling and installing LCDproc you
 may use the support files in the LCDproc source tree to create an
 installation package specific to your distribution
 
-Debian:
-      - Rename the .tgz file with the sources to lcdproc_VERSION.orig.tar.gz
-	  cp lcdproc-VERSION.tgz lcdproc_VERSION.orig.tar.gz
-      - Extract the source tar ball
-	  tar xvzf lcdproc_VERSION.orig.tar.gz
-      - Copy the debian/ directory below scripts/ to the root of the
-	source tree
-	  cp -a lcdproc-VERSION/scripts/debian lcdproc-VERSION/
-      - Generate .deb debian package
-	  cd lcdproc-VERSION/
-	  dpkg-buildpackage -rfakeroot -sd
-	  cd ..
-	This creates a file named lcdproc_VERSION-RELEASE_ARCH.deb
-	that contains the LCDproc suite as a regular Debian package.
+## Debian
+  - Rename the ```.tgz``` file with the sources to ```lcdproc_VERSION.orig.tar.gz```
+```
+cp lcdproc-VERSION.tgz lcdproc_VERSION.orig.tar.gz
+```
+  - Extract the source tar ball
+```
+tar xvzf lcdproc_VERSION.orig.tar.gz
+```
+  - Copy the ```debian/``` directory below scripts/ to the root of the source tree
+```  
+cp -a lcdproc-VERSION/scripts/debian lcdproc-VERSION/
+```
+  - Generate ```.deb``` debian package
+```
+cd lcdproc-VERSION/
+dpkg-buildpackage -rfakeroot -sd
+cd ..
+```
 
```

Comment:
This debian folder doesn't exist anymore. This section is obsolete.

Generally it seems nobody cared about updating the INSTALL file for quite some time...